.github/workflows/ci.yml

```
@@ -2,29 +2,32 @@ name: Backend CI
 
 on:
   push:
-    branches: [ main, develop ]
+    branches: [ '**' ]
   pull_request:
     branches: [ main, develop ]
```

Let's remove develop.  Keep UAT 100% for UAT.

---

I don't understand. Should I add the wildcard to the PR too? Should I remove the wildcard completely from the push? I was trying to enforce code quality even on isolated branches, not only on main or develop, to avoid surprises and fail early.